### PR TITLE
Toml/privacy policy tweaks

### PR DESCRIPTION
**Pull request by lampholder**

### CODE REVIEW

File: docs/matrix-org/privacy_notice.md [hunk 1]
```diff
@@ -56,9 +52,11 @@ Your access and use of the Service is always subject to the most current version
...
 
-New Vector processes your data under *[Legitimate Interest](https://ico.org.uk/for-organisations/guide-to-the-general-data-protection-regulation-gdpr/legitimate-interests/when-can-we-rely-on-legitimate-interests/)*. This means that we process your data only as necessary to deliver the Service, and in a manner that you understand and expect.
+New Vector processes your data under our *[Legitimate Interest](https://ico.org.uk/for-organisations/guide-to-the-general-data-protection-regulation-gdpr/legitimate-interests/when-can-we-rely-on-legitimate-interests/)* to provide our Service to you in a an efficient and secure manner and to ensure the legal compliance and proper administration of our business. Essentially, this means that we process your data only as necessary to deliver the Service and for internal administration purposes,, and in a manner that you understand and expect. We also carry out processing that is necessary to provide our Service to you under our Matrix.org Homeserver Terms and Conditions and processing that is necessary to comply with our legal obligations. Where consent is required by law in relation to certain processing, we will ask for your consent.
```
Comment (ara4n) [inline docs/matrix-org/privacy_notice.md]: s/our business/our project/?

Comment (ara4n) [reply docs/matrix-org/privacy_notice.md]: double ,,

File: docs/matrix-org/privacy_notice.md [hunk 2]
```diff
@@ -56,9 +52,11 @@ Your access and use of the Service is always subject to the most current version
...
+
+We process your information for the purposes of providing our decentralised, openly-federated and (optionally) end-to-end encrypted communication Service, getting in touch with you, responding to your requests, working with our suppliers to deliver the Service and enabling its features, ensuring the security of our Service, developing, fixing and improving our Service, administering our business and complying with the law.
```
Comment (ara4n) [inline docs/matrix-org/privacy_notice.md]: s/business/project/ again?

File: docs/matrix-org/privacy_notice.md [hunk 3]
```diff
@@ -146,19 +146,21 @@ Currently, we log the IP addresses of everyone who accesses the Service. This da
 
 ### 2.3.1 Sharing Data with Connected Services
 
-The {{ policy_homeserver }} homeserver is a *decentralised* and *open* service. This means that, to support communication between users on different homeservers or different messaging platforms, your username, display name and messages and files are sometimes shared with other services that are connected with the {{ policy_homeserver }} homeserver.
+We may share you information when working with our suppliers and the Matrix Foundation to provide the Service.
```
Comment (ara4n) [inline docs/matrix-org/privacy_notice.md]: s/you/your/

File: docs/matrix-org/privacy_notice.md [hunk 4]
```diff
@@ -174,6 +176,12 @@ Access control settings, requests to remove messages by "redactions" or remove p
 
 The {{ policy_homeserver }} homeserver provides a range of integrations in the form of Widgets (miniature web applications accessed as part of a Matrix Client) and Bots (automated participants in rooms). Bots and Widgets currently have access to all the messages and files in any room in which they participate, although we are adding a more sophisticated access control system.
 
+### Transfers of your Data
+
```
Comment (ara4n) [inline docs/matrix-org/privacy_notice.md:330]: UK/EU? matrix.org runs from both

**Comment by ara4n** [comment]
lgtm mod minor tweaks :)

File: docs/matrix-org/privacy_notice.md (hunk 4 above)
Comment (lampholder) [reply docs/matrix-org/privacy_notice.md:330]: I'll switch this to EU for now - it's good 'til Oct 31st at least.

File: docs/matrix-org/privacy_notice.md (hunk 1 above)
Comment (lampholder) [reply docs/matrix-org/privacy_notice.md]: commas fixed; leaving 'business' for now - will seek additional input

File: docs/matrix-org/privacy_notice.md (hunk 2 above)
Comment (lampholder) [reply docs/matrix-org/privacy_notice.md]: Leaving business for now - will confirm

File: docs/matrix-org/privacy_notice.md (hunk 3 above)
Comment (lampholder) [reply docs/matrix-org/privacy_notice.md]: done

**Pull request merged**